package.json

```diff
@@ -102,6 +104,14 @@
     "yoshi": "^4.9.0",
     "yoshi-style-dependencies": "^4.6.0"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
@giladsegal would you consider `pre-push` instead?

Reply:
I was also thinking about that, but i'm not sure.
On the one hand, having it `pre-commit` lets the developer add the lint fixes directly to the commit.
On the other hand, having it in `pre-push` reduces the amount of times linting will run, and might be less of a "pain".